README.rst

```diff
@@ -1,29 +1,100 @@
 skill_tagging
 #############################
 
-.. note::
-
-  This README was auto-generated. Maintainer: please review its contents and
-  update all relevant sections. Instructions to you are marked with
-  "PLACEHOLDER" or "TODO". Update or remove those sections, and remove this
-  note when you are done.
-
 |pypi-badge| |ci-badge| |codecov-badge| |doc-badge| |pyversions-badge|
 |license-badge| |status-badge|
 
-Purpose
-*******
+Overview
+********
+
+Django app for fetching and verifying tags/skills for video and vertical/unit
+XBlocks. It implements two openedx_filters pipelines to inject a form into the end
+unit XBlocks and video XBlocks.
+
+.. image:: https://user-images.githubusercontent.com/10894099/210078679-3cbac3d1-55a7-4fba-b841-7fb4468f32c5.png
+   :target: https://user-images.githubusercontent.com/10894099/210078679-3cbac3d1-55a7-4fba-b841-7fb4468f32c5.png
+   :alt: vertical block verification form
 
-Django app plugin for fetching and verifying tags for xblock skills.
+.. image:: https://user-images.githubusercontent.com/10894099/212285572-efa5cfd5-e9c5-411d-8d15-541c43445ec0.png
+   :target: https://user-images.githubusercontent.com/10894099/212285572-efa5cfd5-e9c5-411d-8d15-541c43445ec0.png
+   :alt: video block verification form
+
+More information about the XBlock skill tagging design can be found in this
+`ADR`_.
+
+.. _ADR: https://github.com/openedx/taxonomy-connector/blob/master/docs/decisions/0001-xblock-skill-tagging-design.rst
 
-Set ``SHOW_SKILL_VERIFICATION_PROBABILITY`` in your django settings to configure
-probability of displaying verification form. Values in range 0 to 1 are
-allowed, where 0 means never and 1 means always display. Default value is 0.5
-i.e. 50% chance of displaying the form.
 
 Getting Started
 ***************
 
+To install ``skill_tagging`` in edx-platform, run
+
+.. code-block::
+
+   pip install skill_tagging
```

Comment:
To install locally we clone a package in src directory and then install from the relevant shell using `pip install -e /edx/src/repo-name`. Can we add this info as well?